### PR TITLE
Deploy/activate uses executor

### PR DESCRIPTION
**Pull request by Naatan**
https://www.pivotaltracker.com/story/show/177635510

### CODE REVIEW

File: test/integration/deploy_int_test.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 	"time"
 
+	executor2 "github.com/ActiveState/cli/pkg/platform/runtime/executor"
```
Comment (mdrohmann) [inline test/integration/deploy_int_test.go:12]: The `executor2` should probably be removed now...

**Review by mdrohmann — CHANGES_REQUESTED** [review]
Looks good.  Just very minor things that I found.

**Review by mdrohmann — DISMISSED** [review]

File: internal/appinfo/appinfo.go [hunk 2]
```diff
@@ -17,14 +17,14 @@ type AppInfo struct {
 func TrayApp() *AppInfo {
 	return &AppInfo{
 		constants.TrayAppName,
-		absolute(filepath.Join(filepath.Dir(os.Args[0]), "state-tray") + osutils.ExeExt),
+		exePath(filepath.Join(filepath.Dir(os.Args[0]), "state-tray") + osutils.ExeExt),
```
Comment (mdrohmann) [inline internal/appinfo/appinfo.go]: I noticed that this needs to look like this now:

 ```
exePath("state-tray")
```

File: internal/appinfo/appinfo.go [hunk 3]
```diff
@@ -17,14 +17,14 @@ type AppInfo struct {
...
 	}
 }
 
 func StateApp() *AppInfo {
 	return &AppInfo{
 		constants.StateAppName,
-		absolute(filepath.Join(filepath.Dir(os.Args[0]), "state") + osutils.ExeExt),
+		exePath(filepath.Join(filepath.Dir(os.Args[0]), "state") + osutils.ExeExt),
```
Comment (mdrohmann) [inline internal/appinfo/appinfo.go]: same here:
```
exePath("state")
```

**Review by mdrohmann — COMMENTED** [review]
I think it is this problem!

File: internal/appinfo/appinfo.go (hunk 2 above)
Comment (Naatan) [reply internal/appinfo/appinfo.go]: Thanks, just noticed that also -.-

**Pull request merged**